exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/LateralJoinBatch.java

```diff
@@ -491,6 +482,7 @@ private IterOutcome processLeftBatch() {
               "received NOT_YET");
           }
           break;
+        default:
```

Comment:
nit: here and below: can you indicate `fall through` with a comment in `default` cases, please?